Header header
int32 x
int32 y
float32 velocityRatio
float32 accelerationRatio